### DISCUSSION TITLE
Please help me with contributions (Anime List)

### DISCUSSION THREAD

**Discussion by raux** [NAIST-SE/AttractingContributors#4] [General]
I need help with my anime list. 

To help you, please look at this list
https://www.kaggle.com/azathoth42/myanimelist

Come to my fork:
https://github.com/raux/AttractingContributors-Anime

Add to my README file. 

I look forward to your contributions

**Comment by raux** [comment]
I submitted a Pull Request!

**Comment by nosfill** [comment]
I have sent a PR. I hope you like the anime I have added.

**Comment by zhangyongxin1212** [comment]
Added, plz help me too
https://github.com/zhangyongxin1212/AttractingContributors-Game

**Comment by MatsumuraTaiki** [comment]
Added, please help me too
https://github.com/MatsumuraTaiki/AttractingContributors-Actors/edit/main/README.md

**Comment by KazukiSenda** [comment]
Added my favorite one !

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter
